pandas/core/generic.py

```diff
@@ -3521,6 +3519,14 @@ def interpolate(self, method='linear', axis=0, limit=None, inplace=False,
         dtype: float64
 
         """
+
+    @Appender(_shared_docs['interpolate'] % _shared_doc_kwargs)
+    def interpolate(self, method='linear', axis=0, limit=None, inplace=False,
+                    limit_direction='forward', downcast=None, **kwargs):
+        """
+        .. versionadded:: 0.18.1
+            ``.resample(..).interpolate()`` is now supported (:issue:`12925`)
```

Comment:
Putting this here in the beginning of the docstring does not really work well I think. Because the rest (the `shared_docs['interpolate']`) are appended, the explanation starts with 'versionadded', while it should rather be "Interpolate values according to different methods" (although, as long as it is not listed in an API overview, it is not really that big of a problem. But, it should actually be added here:http://pandas-docs.github.io/pandas-docs-travis/api.html#upsampling)